src/Url.js

```diff
@@ -52,7 +52,7 @@ const Url = React.createClass({
 
   onClick(event) {
     event.preventDefault();
-    ActionCreator.navigateTo(this.props);
+    Store.dispatch(ActionCreator.navigateTo(this.props));
```

Comment:
I'm not quite sure here. 
Is it a usual convention in Redux apps to apply changes to store from the component? I liked Flux because it actually gave me this independence:

`component` -> `action creator` -> `dispatcher` -> `store` -> `component`

So each item had only one input and one output, never "duplex".